src/server/ftpserver/options.rs

```diff
@@ -216,3 +216,45 @@ impl Default for Shutdown {
         }
     }
 }
+
+#[derive(Debug, Clone)]
+/// Variants for failed logins protection policy
+pub enum FailedLoginsPolicy {
+    /// User plus source IP address locking
+    SourceUserLock(FailedLoginsPenalty),
+    /// Source IP locking
+    SourceLock(FailedLoginsPenalty),
```

Comment:
`SourceIPLock` perhaps as name?

Nice options...

Reply:
Should we also have Off as an option?

Reply:
Not necessary, the failed logins cache is an option in itself that is disabled by default. Well currently it's on by default but that was not how it is supposed to be. So which is what I'm going to fix next.

Reply:
Default is now None